compiler/modules/nback/compare.bal

```diff
@@ -47,6 +47,15 @@ final RuntimeFunction booleanCompareFunction = {
     attrs: ["readonly"]
 };
 
+final RuntimeFunction decimalCompareFunction = {
```

Comment:
The _cmp and _compare functions aren't the same. _cmp returns <0, 0, >0 (like strcmp).  The _compare functions return a different set of values that allow for unordered.

The name of the RuntimeFunction variable should always be the same as the runtime function name in camel case.